lib/solidus_graphql_api/context.rb

```diff
@@ -37,7 +37,11 @@ def current_store
     end
 
     def current_pricing_options
-      @current_pricing_options ||= Spree::Config.pricing_options_class.from_context(self)
+      @current_pricing_options ||= if Spree.solidus_gem_version < Gem::Version.new('2.9')
```

Comment:
What do you think to use only the "old implementation" for now, seems to me that `from_context` is a helper that doesn't do much more than calling it.